test/python/converters/test_circuit_to_dag.py

```diff
@@ -41,16 +41,6 @@ def test_circuit_and_dag(self):
         circuit_out = dag_to_circuit(dag)
         self.assertEqual(circuit_out, circuit_in)
 
-    def test_high_degree_determinism(self):
-        """Test that the converter is deterministic in edge order for high-degree nodes."""
-        qr = QuantumRegister(100, "q")
-        cr = ClassicalRegister(100, "c")
-        qc = QuantumCircuit(qr, cr)
-        # This is particularly testing the determinism for "extra" bits that don't appear in the
-        # cargs (aren't used in the circuit body).
-        qc.if_test(expr.equal(cr, 0), QuantumCircuit(qr), qr, [])
-        self.assertTrue(circuit_to_dag(qc).structurally_equal(circuit_to_dag(qc)))
-
```

Comment:
This is an important test - how come it's removed?

Reply:
I think I mentioned this at the time in Slack, but `structurally_equal` is documented as non-working for circuits with control flow, so I'm not sure this test was actually doing anything before ("not sure" only because I haven't looked since I'd noticed the issue). It'd be possible to implement now, I just didn't want to balloon the PR.

Preference on how we handle this?